gammapy/cube/core.py

```diff
@@ -324,6 +320,24 @@ def solid_angle_image(self):
         sky_map = SkyMap.from_image_hdu(image_hdu)
         return sky_map.solid_angle()
 
+    def sky_image(self, idx_energy, copy=True):
+        """Slice 3-dim cube into a 2-dim `~gammapy.image.SkyMap` along slicepos.
```

Comment:
The `SkyCube.sky_image` method was merged in #586, I can now see it in master:
https://github.com/gammapy/gammapy/blob/master/gammapy/cube/core.py#L324

I don't know why it still shows up in the diff for this PR on Github!?

@OlgaVorokh Can you please rebase this PR and this will refresh the diff?